doc/design/test-dandisets-1.md

```diff
@@ -0,0 +1,111 @@
+only meant for exploration and development.
+
+Due to the relative ease of creating a dandiset (to check out the system),
+compared to curating a dandiset for actual work, the production instance has
+more such test dandisets in it than "real" ones. This in turn degrades the
+perception of the archive as a serious place to host data, and so we need some
+way to (1) discourage the creation and abandonment of exploratory dandisets; (2)
+mark dev dandisets in some way to prevent them from appearing in the main
+listing; and (3) clean up the archive from such abandoned dandisets.
+
+## Proposals
+
+### "Test" flag
+
+This proposes to add a `testing` flag to all dandisets. At creation time, a
+dandiset can optionally have this flag set to prevent it from showing up in
+public listings of all dandisets. These would only show up in the owners' "my
+dandisets" listings (or in all listings shown to the owners). Admins would get a
+checkmark to toggle the display of test dandisets, to help simulate what other
+users are seeing when they visit the archive.
+
+To encourage the use of this feature, the dandiset creation screen would have a
+checkbox along with a description of what this is for ("If you are new to the
+archive and are just exploring, please consider using the staging server
+instead. Otherwise, select this option to avoid this dandiset from appearing in
+public searches.", with "staging server" linked thereto).
+
+### Hide draft-only dandisets
+
+Similarly to hiding explicitly marked test dandisets, we could, by default, also
+hide dandisets that have no published versions (as above, these dandisets would
+still appear for the owners). As the public may *want* to see draft-only
+dandisets, everyone would be able to toggle this mode with a setting.
+
+**NOTE: this feature has already been implemented.**
+
+### Hide dandisets with no assets
+
+Another variation on the previous proposal: hide dandisets that contain no
+assets. These dandisets are a subset of draft-only dandisets, since publishing a
+dandiset requires assets to be present.
+
+This option is problematic on its own--it is a valid use case to declare
+scientific intentions by creating a draft dandiset with no assets, then collect
+and add data only after months or years of real scientific work. That is to say,
+an assetless dandiset that nonetheless has a description, funding agent,
+contact person, owners, etc. may indeed be fully valid.
+
+**NOTE: this feature has already been implemented.**
+
+### Add questions to dandiset creation page
+
+Adding questions such as:
+- Do you welcome collaborators? (Y/N)
+- When do you anticipate posting data to the dandiset?
+
+may help to stem the tide of abandoned dandisets.
+
+The collaborators question, answered positively, would add a badge to the
+dandiset's landing page and search result display indicating the call for
+collaboration.
+
+The anticipation date question would delay considering the dandiset abandoned
+until after that date.
```

Comment:
This seems like an awkward and indirect way to prevent the problem we're addressing here. I move to strike this section from the design.